fiftyone/core/spaces.py

```diff
@@ -16,9 +23,11 @@ def _validate_children(children):
 
 
 class Component(EmbeddedDocument):
+    """Base class for App components."""
 
     meta = {"abstract": True, "allow_inheritance": True}
 
+    # @todo remove
```

Comment:
We can just remove this `id` prop, right? Deprecated by `component_id` I believe

Reply:
Yes, will remove 👍